pkgs/servers/imgproxy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub, pkg-config, vips, gobject-introspection }:
+
+buildGoModule rec {
+  version = "2.8.1";
```

Comment:
Woops, forgot this one: Please move `pname` above `version`.

Reply:
Fwiw, I've stopped complaining about tiny issues like this, since I've been told there's too much bikeshedding like that, which can be demotivating for contributors.